compiler/src/java_plugin/cpp/java_generator.cpp

```diff
@@ -670,10 +679,11 @@ static void PrintService(const ServiceDescriptor* service,
   (*vars)["service_name"] = service->name();
   (*vars)["file_name"] = service->file()->name();
   (*vars)["service_class_name"] = ServiceClassName(service);
+  (*vars)["grpc_version"] = XSTR(GRPC_VERSION);
```

Comment:
@carl-mastrangelo, what is unacceptable internally? It seems `unknown` is just fine internally, or we could change it to `nothing to see here, move along` with a define in our build.